build.gradle

```diff
@@ -0,0 +1,97 @@
+import java.nio.file.Files
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING
+
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'maven-publish'
+
+group 'org.logstash.plugins.input.http'
+version "${new File("VERSION").text.trim()}"
+
+description = "HTTP Input Netty implementation"
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.hamcrest:hamcrest-library:1.3'
+    testCompile 'org.apache.logging.log4j:log4j-core:2.6.2'
+
+    compile 'io.netty:netty-all:4.1.3.Final'
+    compile 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork23'
+    compile 'org.javassist:javassist:3.20.0-GA'
+    compile 'com.fasterxml.jackson.core:jackson-core:2.7.5'
+    compile 'com.fasterxml.jackson.core:jackson-annotations:2.7.5'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.7.5'
+    compile 'com.fasterxml.jackson.module:jackson-module-afterburner:2.7.5'
```

Comment:
what are the jackson and javaassist dependencies for ?

Reply:
copy paste nonsense.